README.md

```diff
@@ -22,46 +24,48 @@ Welcome to Docs! The open source document editor where your notes can become kno
 
-## Why use Docs ❓
-
-⚠️ **Note that Docs provides docs/pdf exporters by loading [two BlockNote packages](https://github.com/suitenumerique/docs/blob/main/src/frontend/apps/impress/package.json#L22C7-L23C53), which we use under the AGPL-3.0 licence. Until we comply with the terms of this license, we recommend that you don't run Docs as a commercial product, unless you are willing to sponsor [BlockNote](https://github.com/TypeCellOS/BlockNote).**
+## Why use Docs?
 
 Docs is a collaborative text editor designed to address common challenges in knowledge building and sharing.
 
 ### Write
-*   😌 Simple collaborative editing without the formatting complexity of markdown
-*   🔌 Offline? No problem, keep writing, your edits will get synced when back online
-*   💅 Create clean documents with limited but beautiful formatting options and focus on content
-*   🧱 Built for productivity (markdown support, many block types, slash commands, keyboard shortcuts).
-*   ✨ Save time thanks to our AI actions (generate, sum up, correct, translate)
-
-### Collaborate
-*   🤝 Collaborate with your team in real time
-*   🔒 Granular access control to ensure your information is secure and only shared with the right people
-*   📑 Professional document exports in multiple formats (.odt, .doc, .pdf) with customizable templates
-*   📚 Built-in wiki functionality to turn your team's collaborative work into organized knowledge `ETA 02/2025`
+*   😌 Get simple, accessible online editing for your team.
+*   💅 Create clean documents with beautiful formatting options.
+*   🖌️ Focus on your content using either the in-line editor, or [the Markdown syntax](https://www.markdownguide.org/basic-syntax/).
+*   🧱 Quickly design your page thanks to the many block types, accessible from the `/` slash commands, as well as keyboard shortcuts.
+*   🔌 Write offline! Your edits will be synced once you're back online.
+*   ✨ Save time thanks to our AI actions, such as rephrasing, summarizing, fixing typos, translating, etc. You can even turn your selected text into a prompt!
+
+### Work together
+*   🤝 Enjoy live editing! See your team collaborate in real time.
+*   🔒 Keep your information secure thanks to granular access control. Only share with the right people.
+*   📑 Export your content in multiple formats (`.odt`, `.docx`, `.pdf`) with customizable templates.
+*   📚 _Coming soon! Built-in wiki functionality to turn your team's collaborative work into organized knowledge._
+
+> ⚠️ **Note: The docx/pdf exporters rely on [packages Blocknote project](https://github.com/suitenumerique/docs/blob/main/src/frontend/apps/impress/package.json#L22C7-L23C53), which are under the AGPL-3.0 licence. Because the Docs project uses the MIT license, we recommend that you do not run Docs as a commercial product -- unless you are willing to sponsor [BlockNote](https://github.com/TypeCellOS/BlockNote). We are working on complying with the terms of the Blocknote license.**
```

Comment:
Hey, I've reworked the phrasing for this.
`Note that the Docs export feature (docx, odt, pdf) relies on [two BlockNotejs packages](https://github.com/suitenumerique/docs/blob/main/src/frontend/apps/impress/package.json#L22C7-L23C53), which are dual-licensed as AGPL-3.0 or proprietary license by Blocknotejs. Please ensure to fulfill your obligation regarding BlockNoteJS licensing (see https://github.com/TypeCellOS/BlockNote/blob/main/packages/xl-pdf-exporter/LICENSE and https://www.blocknotejs.org/about#partner-with-us).`